backend/papersfeed/utils/likes/utils.py

```diff
@@ -127,6 +137,14 @@ def insert_like_review(args):
     review = Review.objects.get(id=review_id)
     review_author = User.objects.get(id=review.user_id)
 
+    # store an action for subscription feed
+    req_user = User.objects.get(id=request_user.id)
```

Comment:
This is redundant. request_user is already what you want.

Reply:
fixed